src/com/example/M3/Task4.java

```diff
@@ -0,0 +1,24 @@
+package com.example.M3;
+import java.util.Scanner;
+
+public class Task4 {
+    public static void main(String[] args) {
+        createForms();
+    }
+
+    public static void createForms() {
+        Scanner scanner = new Scanner(System.in);
+
+        String firstName = scanner.nextLine().toUpperCase();
+        String lastName = scanner.nextLine().toUpperCase();
```

Comment:
better to transform to uppercase later.
```suggestion
        String firstName = scanner.nextLine();
        String lastName = scanner.nextLine();
        firstName = firstName.toUpperCase();
        lastName = lastName.toUpperCase();
```